reference/motion.md

```diff
@@ -149,11 +149,11 @@ motion file in milliseconds.
 The `wbu_motion_get_time()` function returns the current playback position in
 milliseconds.
 
-The `wbu_motion_set_time()` function allows to change the playback position.
-This enables, for example, to skip forward or backward. Note that, the position
+The `wbu_motion_set_time()` function allows to change the playback position in time.
+This allows the user to skip forward or backwards. Note that, the position time
```

Comment:
I would also change ``forward`` in ``forwards`` to be consistent with ``backwards``.